### PR TITLE
[ADD] gap_analysis_price

### PR DESCRIPTION
**Pull request by xie8899**

### CODE REVIEW

File: gap_analysis_price/README.rst [hunk 1]
```diff
@@ -0,0 +1,79 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+{gap_analysis_price}
+==============
+
+have price/cost estimation for business requirements.
```
Comment (elicoidal) [inline gap_analysis_price/README.rst]: This module extends the gap analysis module set to allow
price estimation for the analysis, based on the time estimation and 
cost structure (resource, task type and price).

File: gap_analysis_price/README.rst [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+Installation
+============
+
+To install this module, you need to:
+
```
Comment (elicoidal) [inline gap_analysis_price/README.rst]: No specific steps required for installation

File: gap_analysis_price/README.rst [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+
+Configuration
+=============
+
+To configure this module, you need to:
+
```
Comment (elicoidal) [inline gap_analysis_price/README.rst]: - Create a cost structure in the Gap analysis/Configuration/Cost Structure.
- Specify the cost structure at the CRM opportunity level.

File: gap_analysis_price/README.rst [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+* ...
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/
+{project_repo}/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback `here <https://github.com/OCA/
+{project_repo}/issues/new?body=module:%20
```
Comment (elicoidal) [inline gap_analysis_price/README.rst]: Modify accordingly

**Comment by rafaelbn** [comment]
This module depends in https://github.com/OCA/project/pull/127

**Pull request closed without merging**